pkg/metadata/inventories_collector.go

```diff
@@ -50,7 +50,11 @@ func (c inventoriesCollector) Send(ctx context.Context, s *serializer.Serializer
 // all agents that wish to track inventory, after configuration is initialized.
 func (c inventoriesCollector) Init() error {
 	inventories.InitializeData()
-	return inventories.StartMetadataUpdatedGoroutine(c.sc, time.Duration(config.Datadog.GetInt("inventories_min_interval"))*time.Second)
+	minInterval := time.Duration(config.Datadog.GetInt("inventories_min_interval")) * time.Second
+	if minInterval == 0 {
+		minInterval = config.DefaultInventoriesMinInterval
```

Comment:
This seems like a good thing to have a warning about in the log, same for the one below

Reply:
There's already a warning from Viper -- this is just doing something reasonable in that case.

Reply:
> There's already a warning from Viper

Fair as long as the warning is specifically mentioning the field in question